schemas/question/1.0/schema.json

```diff
@@ -20,6 +20,15 @@
     "description": {
       "type": "string"
     },
+   "processId": {
+            "type": "string"
+        },
+   "questionId": {
```

Comment:
What is `questionId`? Is it same as identifier of the question object? If so, we don't need this right.

Reply:
@maheshkumargangula QuestionId Is a Reference for the users to reach to the appropriate question when they find the status of the bulk upload. 
We have renamed the filed name from QuestionId to QuestionFileRefId, to avoid more functions. 

Also, as suggested by Karteek, We are raising a PR in creation-portal repo [https://github.com/Sunbird-Ed/creation-portal/tree/release-4.2.0/kp_schemas]